builder/package.ubuntu-1804

```diff
@@ -0,0 +1,50 @@
+if [[ ! -d /tmp/output/ubuntu-1804 ]]; then
+  mkdir /tmp/output/ubuntu-1804
+fi
+
+fpm \
```

Comment:
Do we need to add package metadata like license, description, maintainer? If it's coming later, feel free to ignore

Reply:
✅ Done - I've used the repo url in place of the "Maintainer"